src/atc/cosmos/cosmos.py

```diff
@@ -12,14 +12,21 @@
 from typing import Union
 
 from azure.cosmos import CosmosClient
-from pyspark.sql import DataFrame
+from pyspark.sql import DataFrame, types
 from pyspark.sql.types import DataType
 
+from atc.atc_exceptions import AtcException
 from atc.config_master import TableConfigurator
+from atc.cosmos.cosmos_base_server import CosmosBaseServer
+from atc.cosmos.cosmos_handle import CosmosHandle
 from atc.spark import Spark
 
 
-class CosmosDb:
+class AtcCosmosException(AtcException):
+    pass
+
+
+class CosmosDb(CosmosBaseServer):
     def __init__(
         self,
```

Comment:
Why dont use have a ```self.db = self.client.get_database_client(self.database)``` in your ```__init__```? It seems like you refer to it in several methods the exact same way.

Reply:
well spotted. I added it.